src/faebryk/core/cpp/CMakeLists.txt

```diff
@@ -38,6 +38,8 @@ message(STATUS "Python_EXECUTABLE: ${Python_EXECUTABLE}")
 message(STATUS "Python_INCLUDE_DIR: ${Python_INCLUDE_DIR}")
 message(STATUS "Python_LIBRARY: ${Python_LIBRARY}")
 
+#set(CMAKE_BUILD_TYPE "Debug")
```

Comment:
nit; Make this an arg on input, environment var etc... Is this meant to remain a hanging comment?

Reply:
it already is a CMake arg